shared/actions/tracker.js

```diff
@@ -626,8 +637,19 @@ function serverCallMap (dispatch: Dispatch, getState: Function, skipPopups: bool
           },
         })
 
+        // Doing a non-tracker so explicitly cleanup instead of using the timeout
+        if (skipPopups) {
```

Comment:
if its an incoming call (tracker) we actually let the timeout happen so we have some time where we ignore incoming requests.